app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/summary/SummaryProductFragment.kt

```diff
@@ -498,6 +518,18 @@ class SummaryProductFragment : BaseFragment(), ISummaryProductPresenter.View {
                 // showCategoryPrompt true
                 binding.addNutriscorePrompt.text = getString(R.string.add_category_prompt_text)
             }
+            showEcoscorePrompt -> {
+                // showEcoscorePrompt true
+                binding.addNutriscorePrompt.text = getString(R.string.couldnot_compute_ecoscore_prompt_text)
```

Comment:
This should be a button if I'm right. That's a lot of text for a button. Could we maybe get it to something like a line of text?

Reply:
yes, it is a button. 
@teolemon maybe we can shorten the text ?

Reply:
@teolemon